UserDetailsClient/UserDetailsClient.Core/MainPage.xaml.cs

```diff
@@ -5,40 +5,38 @@
 using System.Text;
 using Microsoft.Identity.Client;
 using Newtonsoft.Json.Linq;
+using UserDetailsClient.Core.Features.LogOn;
 using Xamarin.Forms;
 
 namespace UserDetailsClient.Core
 {
     public partial class MainPage : ContentPage
     {
+        protected static readonly IAuthenticationService AuthenticationService;
```

Comment:
should not be needed to make this static

Reply:
it was originally static, i did minimal code touches to reorganize outside of App.xaml.cs.

I eliminated static usage though.